docs/en/core/defaults-files.md

```diff
@@ -81,17 +81,60 @@ variables:
 
-#### Reference Document 
+#### Producing the Reference Document
 
-Use an existing Word document or download the reference.docx from Pandoc and change the styles. Your reference doc is a Word file that you format to your preferences. You need to adjust the Word Styles (Paragraph, Heading, Body Text, Footnote Text, etc) for it to apply. 
+For best results, it is recommended to download the reference document directly from Pandoc. It should already be installed with Zettlr, but if not, see [Pandoc installation](https://docs.zettlr.com/en/installing-pandoc/) instructions. 
+
+##### Option 1: Using Pandoc from Command line or Terminal 
+
+In the command line: `pandoc -o "custom-reference.docx" --print-default-data-file reference.docx`
+
+To adjust the name or location of where the document is saved, you can change `"custom-reference.docx"` to include the absolute path: `"/Users/[user]/Downloads/project-reference.docx"`
+
+##### Option 2: Download from Pandoc Github repository
+
+ In the Pandoc repo the documents are unzipped, which makes downloading them somewhat complicated. 
+
+1. Open https://github.com/jgm/pandoc/
+2. In the green Code button, select "Download zip"
+3. Open the folder, find /pandoc-master/data/docx, and ZIP (compress) the docx folder. 
+4. Change ".zip" to ".docx" in the file name
+5. Open with Word
+
+This method can lead to errors, and it is recommended to follow Option 1. 
+
+##### Option 3: Finding documents produced by others
+
+Formatted Pandoc reference documents can be found on Github or elsewhere. Use these at your own risk. 
+
+Examples: 
+- https://github.com/hokorobi/pandoc-docx-reference/blob/master/reference.docx
+- https://www.autodidacts.io/convert-markdown-to-standard-manuscript-format-odts-docs-and-pdfs-with-pandoc/ 
+
+
+#### Adjusting the style and format
+
+The reference document is a Word file that needs to be formatted to your preferences using Styles. 
+<img width="500" alt="image" src="https://user-images.githubusercontent.com/109297886/200125385-1894484c-ea68-4615-aca3-065e40d271d9.png">
+
+You must update the formatting in the Styles Pane (Paragraph, Heading 1, Body Text, Footnote Text, etc.) for it to apply. Changing the text or other contents of the document has no effect unless you "Update to Match Selection" in Styles. Pandoc ONLY looks at the Styles format. 
+
+- More information about which Styles are supported in Pandoc [documentation](https://pandoc.org/MANUAL.html#options-affecting-specific-writers) under `--reference-doc=FILE`. 
+
+If you have an existing document that you want to use as template, you can use Word Organizer to copy the styles over. 
+ 
+1. Format > Style...
+2. Organizer... at bottom left
+3. On the right side, under `Normal.dotm (global template)`, close file
+4. Open file, find your desired document
+5. Select the styles on the right, and <- Copy it over to the current reference.docx
```

Comment:
Is the `<-` arrow accidental?

Reply:
No, I was trying to describe that the arrow should be pointing to the left so that you're copying it over in the correct direction, but maybe I should just add another screenshot? But it might look different in windows than mac.  
<img width="691" alt="image" src="https://user-images.githubusercontent.com/109297886/200150542-daad102e-4a6b-4705-9e0c-a06303d476a6.png">

(I'll take a better screenshot and upload it properly once I figure out how